samples/ethnography05.json

```diff
@@ -0,0 +1,15 @@
+{
+  "_id": "8b6367c9171b8b1e44625e035c0062aa",
+  "links": [
+    {
+      "verb": "isGroundedIn",
+      "object": "6b56ee657c870dfacd34e9ae4e0643dd"
+    }
+  ],
+  "text": "{1} \n {2} Se socialiser\nDonner son avis sur une tâche en cours\n {3} \n {4} Se concerter (lors de la pause du midi)",
```

Comment:
With this version, you didn't "grounded" the code precisely in the text. Please note that: 
- to get a precise anchor you need both the block identifier and the quoted text,
- as stated in [documentation](https://github.com/Hypertopic/HyperGlosae/blob/6a1b4a56dea8d3d0fb2a3de3e49d972c503b3848/samples/inrap_D0.json#L12), the quoted text should use [Markdown definition lists](https://www.markdownguide.org/extended-syntax/#definition-lists).

Please test with the following `text`:

```json
"{2} Se socialiser\n: plusieurs personnes se présentent à moi. Ayant identifié que je suis nouveau, elles me souhaitent la bienvenue\n\nDonner son avis sur une tâche en cours\n: Un autre m'indique que j'aurais intérêt à commencer par les parties hautes de la locomotive, plutôt que des côtés, car celles-ci deviendront vite très chaudes (je risque de me bruler si j'attends pour les laver){4} Se concerter (lors de la pause de midi)\n: Luc et Leo exposent les résultats des essais. Les différentes personnes présentes discutent des solutions possibles"
```